.travis.yml

```diff
@@ -1,4 +1,20 @@
 language: generic
+matrix:
+  include:
+    - name: "generic tests"
+    - name: "flake8 lint on Python 2.7"
+      language: python
+      python: 2.7
+      install: pip install flake8
+      script: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+    - name: "flake8 lint on Python 3.7"
+      dist: xenial  # required for Python >= 3.7
+      language: python
+      python: 3.7
+      install: pip install flake8
+      script: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+  allow_failures:
+    - python: 3.7
```

Comment:
Why allow failures for Python 3.7?

Reply:
The current code contains Python 3 syntax errors and Travis CI [__allow failures__ _lets you add in experimental and preparatory builds to test against versions or configurations that you are not ready to officially support_](https://docs.travis-ci.com/user/build-matrix/#rows-that-are-allowed-to-fail).

Reply:
Shouldn't we fix those syntax errors instead? /cc @yonghong-song